tornado-benchmarks/src/main/java/uk/ac/manchester/tornado/benchmarks/dgemm/DgemmTornado.java

```diff
@@ -72,39 +69,11 @@ public void setUp() {
         }
 
         taskGraph = new TaskGraph("benchmark");
-        if (!USE_PREBUILT) {
-
-            taskGraph.transferToDevice(DataTransferMode.EVERY_EXECUTION, a, b) //
-                    .task("dgemm", LinearAlgebraArrays::dgemm, m, n, n, a, b, c) //
-                    .transferToHost(DataTransferMode.EVERY_EXECUTION, c);
-
-            immutableTaskGraph = taskGraph.snapshot();
-            executionPlan = new TornadoExecutionPlan(immutableTaskGraph);
-            executionPlan.withWarmUp();
-
-        } else {
-            String filePath = "/tmp/mxmDouble.spv";
-            TornadoDevice device = null;
-            int maxDevices = TornadoRuntimeProvider.getTornadoRuntime().getBackend(0).getNumDevices();
-            for (int i = 0; i < maxDevices; i++) {
-                device = TornadoRuntimeProvider.getTornadoRuntime().getBackend(0).getDevice(i);
-                if (device.isSPIRVSupported()) {
-                    break;
-                }
-            }
-
-            taskGraph.transferToDevice(DataTransferMode.EVERY_EXECUTION, a, b) //
-                    .prebuiltTask("t0", //
-                            "dgemm", //
-                            filePath, //
-                            new Object[] { m, n, n, a, b, c }, //
-                            new Access[] { Access.READ_ONLY, Access.READ_ONLY, Access.READ_ONLY, Access.READ_ONLY, Access.READ_ONLY, Access.WRITE_ONLY }, //
-                            device, //
-                            new int[] { n, n })//
-                    .transferToHost(DataTransferMode.EVERY_EXECUTION, c); //
-            immutableTaskGraph = taskGraph.snapshot();
-            executionPlan = new TornadoExecutionPlan(immutableTaskGraph);
-        }
+        taskGraph.transferToDevice(DataTransferMode.EVERY_EXECUTION, a, b) //
+                .task("dgemm", LinearAlgebraArrays::dgemm, m, n, n, a, b, c) //
+                .transferToHost(DataTransferMode.EVERY_EXECUTION, c);
+        executionPlan = new TornadoExecutionPlan(taskGraph.snapshot());
+        executionPlan.withWarmUp();
```

Comment:
in this case, we removed completely the prebuilt API usage for this benchmark?

Reply:
Yes, that was for an experiment with did for Intel, but do not need to keep this in this version.